services/prefect/flows/generated_flows.py

```diff
@@ -75,9 +77,13 @@ def initialize_sentry(sentry_dsn: str):
 
 
 @task
-def skip_cached_flow(flow_name):
-    logger = prefect.context.get("logger")
-    logger.info(f"No new data. Skipping {flow_name}")
+def skip_cached_flow():
```

Comment:
Thanks!  As an aside, whenever you write a long comment on a github code review it's probably worth asking if part(s) of it should be in a comment directly in the code instead.

In this case, the code comment is pretty good, but explicitly referencing `etag_caching_terminal_state_handler` instead of just "The flow's terminal state handler" might be helpful to somebody coming along later and trying to understand this code.

Reply:
Good point, and I've added a reference to etag_caching_terminal_state_handler in 629ede449fdf370211f6e549ce4c1915c8c378e1